photon-ml/src/main/scala/com/linkedin/photon/ml/cli/game/training/Params.scala

```diff
@@ -25,12 +25,46 @@ import com.linkedin.photon.ml.supervised.TaskType._
 
 
 /**
+ * Command line arguments for GAME
+ * @param trainDirs Input directories of training data. Multiple input directories are also accepted if they are
+ *                  separated by commas, e.g., inputDir1,inputDir2,inputDir3.
+ * @param trainDateRangeOpt Date range for the training data represented in the form start.date-end.date,
+ *                          e.g. 20150501-20150631. If trainDateRangeOpt is specified, the input directory is expected
+ *                          to be in the daily format structure (e.g., trainDir/daily/2015/05/01), if not, then no
+ *                          assumption is on the structure of the input directory.
```

Comment:
Good point. Your understanding is correct, but please let me know your suggestion on how to improve the wording to make it less confusing here.